### DISCUSSION TITLE
Contributing translations is now much easier!

### DISCUSSION THREAD

**Discussion by jcorporation** [jcorporation/myMPD#803] [Announcements]
I imported the existing myMPD translations (also the very outdated) to POEditor as a public project. Contributing to translation should be now much easier.

Join the translation project: https://poeditor.com/join/project/Z54inZwdul

New contributors require moderation, please post here a short notification to notify me. 

Documentation of the translation framework: https://jcorporation.github.io/myMPD/060-references/translating.html

**Note:** There are some phrases that are only shortcuts for the full text. You can lookup the English translation here: https://github.com/jcorporation/myMPD/blob/master/src/i18n/json/en-US.json

**Comment by jccricket** [comment]
Hello,

I would like to try to contribute to French translation.
I'm registered in poeditor. I guess this should trigger a request of something.
Anyway, as per your guidance, here is a message to tell you.

JC

**Reply by jcorporation** [reply]
I approved your request. Thanks for contributing to myMPD!

**Reply by tsunulukai** [reply]
Is there some kind of limitation on the number of languages one can contribute to on POEditor ?
I registered both for the dutch and french translations, but every time I get activated for one, I'm getting kicked out for the other.

**Reply by jcorporation** [reply]
I do not know any limitations, but the contributors page is strange

![image](https://user-images.githubusercontent.com/765844/189543247-e6795b03-c246-4614-8d90-e0cac00f40f3.png)

I kick all the 3 requests, please try again.

**Reply by jcorporation** [reply]
I approved you again, happy translating!

**Reply by tsunulukai** [reply]
I heavily reworked the french translations and tried to standardize style across different strings/phrases.
@jcorporation could you merge it into devel ?

**Reply by jcorporation** [reply]
Merged! Thanks for your translation efforts and bug reporting!

**Comment by DoDiesis** [comment]
Some time ago I contributed with the italian translation, now I would go on reviewing the new additions.

**Reply by jcorporation** [reply]
I approved your contribution request on poeditor.com

**Comment by jcorporation** [comment]
v10.1.0 was merged into the devel branch and there are waiting some new phrases for translation ;)

**Reply by DoDiesis** [reply]
What the various strings "mympd_start", "mpd_output", etc. refer to? Can you show a screenshot of the page where they appear?

**Reply by jcorporation** [reply]
This are trigger events.

**Reply by DoDiesis** [reply]
What the "Save mode"string  stands for?
Does it mean: "Save the current mode", or it stase that the current mode is a "saving mode"?

**Reply by DoDiesis** [reply]
> This are trigger events.

So I think I can let them untranslated. Or not?

**Reply by jcorporation** [reply]
I translated it for german.

**Reply by tsunulukai** [reply]
FR/NL updated.

**Reply by jcorporation** [reply]
FR/NL and IT translations are imported

**Reply by DoDiesis** [reply]
New ITA translations added, some older strings corrected not to use different translation for the same string.

**Reply by jcorporation** [reply]
There are again some phrases waiting for translation.

**Reply by DoDiesis** [reply]
> There are again some phrases waiting for translation.

New phrases translated in ITA.

**Reply by jcorporation** [reply]
Updated IT and FR phrases are now imported from POEditor into GitHub. Thanks for translating!

**Reply by tsunulukai** [reply]
NL done too.

**Reply by tsunulukai** [reply]
Added missing FR/NL phrases on POEditor

**Comment by jcorporation** [comment]
There are now new terms for the upcoming 10.2.0 release available.

**Reply by tsunulukai** [reply]
FR/NL Done

**Reply by soya-daizu** [reply]
JA done

**Comment by DoDiesis** [comment]
IT done.

**Comment by jcorporation** [comment]
There are missing some translations for:

- Chinese (simplified)
- Dutch
- French
- Italian
- Korean
- Spanish
- Finish
- Spanish (VE)
- Chinese (traditional)

Any help will be appreciated!

**Reply by tsunulukai** [reply]
FR/NL Done

**Reply by DoDiesis** [reply]
ITA done

**Comment by dream7180** [comment]
Please approved my access to Chinese translation. My POEditor name is dreamawake. Thanks.

**Reply by jcorporation** [reply]
Approved

**Comment by whoami2208** [comment]
Registered on poeditor and submitted for Polish translation - awaiting for approval now!

**Reply by jcorporation** [reply]
Approved, tanks for contributing the polish translation!

**Comment by DoDiesis** [comment]
I just translated the latest missing phrase in Italian.

**Comment by om2804** [comment]
@jcorporation Please approve my access to translation in POEditor, my name name is om.

**Reply by jcorporation** [reply]
Approved, thanks for contributing!

**Reply by om2804** [reply]
@jcorporation RU done
Most likely need adjustments after I see a translation in UI

**Reply by jcorporation** [reply]
Updated

**Reply by om2804** [reply]
@jcorporation RU translation is updated

**Reply by jcorporation** [reply]
Merged

**Comment by jcorporation** [comment]
v10.3.0 source is merged into devel. There are waiting new phrases for translation.

**Reply by soya-daizu** [reply]
🇯🇵👌

**Reply by tsunulukai** [reply]
🇫🇷  / 🇳🇱  👌

**Reply by om2804** [reply]
RU done

**Reply by DoDiesis** [reply]
IT done

**Comment by jcorporation** [comment]
I added another 3 missing phrases.

**Reply by DoDiesis** [reply]
IT done.

**Reply by tsunulukai** [reply]
🇫🇷 / 🇳🇱 👌

**Reply by soya-daizu** [reply]
🇯🇵👌(with other consistency fixes)

**Comment by parkmino** [comment]
Hi,

It's been a while.
Please find the updated Korean translation from POEditor

Thanks

**Reply by jcorporation** [reply]
Merged, thanks!

**Comment by jcorporation** [comment]
There are now some new phrases for the upcoming v11.0.0 release available.

**Reply by soya-daizu** [reply]
🇯🇵👌

**Reply by jcorporation** [reply]
I added another phrase ;)

**Reply by tsunulukai** [reply]
🇫🇷 / 🇳🇱 👌

**Comment by DoDiesis** [comment]
ITA done.

**Comment by ThecaTTony** [comment]
Hi, I would like to improve the es-AR translation, although it is complete it has some errors and inconsistencies. I already applied in POEditor to get access, thank you.

**Reply by jcorporation** [reply]
Thank you! You are approved.

**Comment by jcorporation** [comment]
v12.0.0 source is merged into devel. There are waiting new phrases for translation.

**Reply by ThecaTTony** [reply]
Hi, it seems to me that the script that exported the values to be translated did not work properly and this went to poeditor:

![image](https://github.com/jcorporation/myMPD/assets/6334401/9aab9783-1883-4aaa-bc4c-12b121babcf1)

**Reply by jcorporation** [reply]
This phrases are correct. You can lookup the english translations here: https://github.com/jcorporation/myMPD/blob/devel/src/i18n/json/en-US.json

**Reply by soya-daizu** [reply]
Regarding the need to check en-US.json, I think it has to be mentioned somewhere that some keys such as "helpConnectionMPDHost" are not the actual source strings. I see some other languages have those simply copy-and-pasted without translating the source string.

**Reply by jcorporation** [reply]
Good idea, I add a bit of documentation.

**Reply by soya-daizu** [reply]
BTW, :jp: is ready :ok_hand:

**Reply by jcorporation** [reply]
I imported your translation into devel and updated the start post of this discussion.

**Reply by ThecaTTony** [reply]
es-ar done.

**Reply by tsunulukai** [reply]
🇫🇷 / 🇳🇱 👌

**Reply by tsunulukai** [reply]
Small feedback with regards to the string "Std / Hours ago". In french, we'd need to say "il y a X heures", with X being the number of hours ago, eg: X hours ago. It'd be better if the string to translate contained a variable for the time to account for the fact not all languages express such things in the same wording order.

**Reply by jcorporation** [reply]
This phrase is for:

![image](https://github.com/jcorporation/myMPD/assets/765844/dc9f7f95-dd13-4df1-879a-97593af59997)

**Comment by jcorporation** [comment]
Current translation status:

bg-BG: skipped, too many missing phrases (997)
de-DE: fully translated
en-US: fully translated
es-AR: fully translated
es-ES: skipped, too many missing phrases (795)
es-VE: skipped, too many missing phrases (798)
fi-FI: skipped, too many missing phrases (795)
fr-FR: fully translated
it-IT: skipped, too many missing phrases (155)
ja-JP: fully translated
ko-KR: 3 missing phrases
nl-NL: fully translated
pl-PL: skipped, too many missing phrases (970)
ru-RU: skipped, too many missing phrases (252)
zh-Hans: fully translated

**Reply by om2804** [reply]
Please review RU

**Comment by jcorporation** [comment]
I added some new phrases for the upcoming 12.1.0 release.

**Reply by tsunulukai** [reply]
🇫🇷 / 🇳🇱 👌

**Reply by soya-daizu** [reply]
:jp: :ok_hand:

**Reply by jcorporation** [reply]
You are really fast! Thanks for your continuous translation contributions.

**Reply by ThecaTTony** [reply]
es-ar done.

**Comment by jcorporation** [comment]
New phrases for v13 are now available in poeditor.

**Reply by tsunulukai** [reply]
🇫🇷 / 🇳🇱 👌

**Reply by soya-daizu** [reply]
🇯🇵 👌

**Reply by ThecaTTony** [reply]
es-ar done.

**Comment by DoDiesis** [comment]
🇮🇹 it done.

**Comment by jcorporation** [comment]
Today I had to realize that the GitHub integration of poeditor.com is no more available for my free plan as opensource project. ~Please do not add any translations until I know why.~

Edit: I use now the api

**Comment by jcorporation** [comment]
The v14 branch was merged into devel. There are new terms waiting for translation.

**Reply by tsunulukai** [reply]
🇫🇷 / 🇳🇱 👌

**Reply by DoDiesis** [reply]
🇮🇹 IT done.

**Reply by soya-daizu** [reply]
🇯🇵 partially done. Need more context in the released software to translate the remaining.

**Reply by ThecaTTony** [reply]
es-ar done.

**Comment by skierro** [comment]
Hello, can you approve me in the POEditor fo Polish? My name is Adamo Skierro.

**Reply by jcorporation** [reply]
I can do it not before the weekend. I am on a trip and has no access to the poeditor project.

Thanks for joining the translation team!

**Reply by skierro** [reply]
Hello,
I am trying to import latest Polish translations from poeditor.com locally.
- I have created the API token for my account on poeditor.com
- I have exported token to .secrets (`echo 'export POEDITOR_TOKEN="XXX"' > .secrets`)

Whether I then execute: `./build.sh trans_import_all` or `./build.sh trans_import pl PL-pl`, I get an error: 
`ERROR: Download failed: {"response":{"status":"fail","code":"403","message":"You don't have permission to access this resource"}}`

Can you check the permissions for my account on the poeditor.com?

**Reply by jcorporation** [reply]
I found no settings to add extra rights to your account.

I generated a read only API token: ` 8c68858a12629d612ccd58a432d374f4`

**Reply by skierro** [reply]
Polish language is complete and checked.

**Comment by DoDiesis** [comment]
🇮🇹 IT updated with the last 6 missing strings.

**Reply by jcorporation** [reply]
Great! Merged into devel for 14.0.1 release.

**Comment by extremelyonline** [comment]
Hi, Chinese translation (Both Simplified and Traditional) is now complete

**Comment by jcorporation** [comment]
New terms for the upcoming v15.0.0 release are now waiting for your translation.

**Reply by DoDiesis** [reply]
🇮🇹 IT translation complete.

**Reply by soya-daizu** [reply]
🇯🇵 done.

**Reply by tsunulukai** [reply]
🇫🇷 / 🇳🇱 👌

**Reply by ThecaTTony** [reply]
es-ar done.
Could you put me in touch with the other translator (VoodooYam)? So we agree on some terms, thank you.

**Reply by jcorporation** [reply]
@ThecaTTony: It is ok, if I give him your mail address?

**Reply by ThecaTTony** [reply]
> @ThecaTTony: It is ok, if I give him your mail address?

Sure, thanks.

**Reply by jcorporation** [reply]
I have written a mail to him.

**Comment by jcorporation** [comment]
I merged the v16.0.0 branch into devel and updated poeditor.com. Have fun translating!
Here are the current stats:
```
bg-BG: skipped, too many missing phrases (1052)
de-DE: fully translated
en-US: fully translated
es-AR: 53 missing phrases
es-ES: skipped, too many missing phrases (917)
es-VE: skipped, too many missing phrases (910)
fi-FI: skipped, too many missing phrases (907)
fr-FR: 57 missing phrases
it-IT: 53 missing phrases
ja-JP: 53 missing phrases
ko-KR: 53 missing phrases
nl-NL: 57 missing phrases
pl-PL: 75 missing phrases
ru-RU: skipped, too many missing phrases (148)
zh-Hans: 53 missing phrases
zh-Hant: 75 missing phrases
```

**Reply by soya-daizu** [reply]
🇯🇵 done

**Reply by tsunulukai** [reply]
🇫🇷 / 🇳🇱 👌

**Reply by ThecaTTony** [reply]
es-ar done.

**Reply by DoDiesis** [reply]
🇮🇹 IT done.

**Comment by jcorporation** [comment]
There are new phrases for the upcoming v17.0.0 release ready for translation.

**Reply by tsunulukai** [reply]
🇫🇷 / 🇳🇱 👌

**Reply by ThecaTTony** [reply]
es-ar done.

**Reply by DoDiesis** [reply]
🇮🇹 IT done.

**Reply by soya-daizu** [reply]
🇯🇵 done

**Reply by jcorporation** [reply]
Wow, you are really fast!

**Comment by jcorporation** [comment]
I merged the v18.0.0 branch into devel and updated the [poeditor](https://poeditor.com/projects/view?id=550213) project.

Have fun translating the new terms.

```
bg-BG: skipped, too many missing phrases (1083)
de-DE: fully translated
en-US: fully translated
es-AR: 31 missing phrases
es-ES: skipped, too many missing phrases (950)
es-VE: skipped, too many missing phrases (938)
fi-FI: skipped, too many missing phrases (935)
fr-FR: 31 missing phrases
it-IT: 31 missing phrases
ja-JP: 31 missing phrases
ko-KR: 31 missing phrases
nl-NL: 31 missing phrases
pl-PL: 83 missing phrases
ru-RU: 31 missing phrases
zh-Hans: 31 missing phrases
zh-Hant: skipped, too many missing phrases (117)
```

**Reply by soya-daizu** [reply]
🇯🇵 done

**Reply by tsunulukai** [reply]
🇫🇷 / 🇳🇱 👌

**Reply by ThecaTTony** [reply]
es-ar done.

You have a typo error in https://github.com/jcorporation/myMPD/blob/devel/src/i18n/json/en-US.json#L111

**Reply by jcorporation** [reply]
Thanks for the hint.

**Comment by DoDiesis** [comment]
🇮🇹 Latest 5 Italian translations, found in version 18.1.2, added.

**Reply by tsunulukai** [reply]
Idem for 🇫🇷 / 🇳🇱

**Reply by ThecaTTony** [reply]
Same for es-ar.

**Comment by DoDiesis** [comment]
🇮🇹 Latest new Italian translation, found in version 18.2.0, added.

**Comment by jcorporation** [comment]
I merged the v18.0.0 branch into devel and updated the [poeditor](https://poeditor.com/projects/view?id=550213) project.

Have fun translating the new terms.

```
bg-BG: skipped, too many missing phrases (1090)
de-DE: fully translated
en-US: fully translated
es-AR: 1 missing phrases
es-ES: skipped, too many missing phrases (957)
es-VE: skipped, too many missing phrases (945)
fi-FI: skipped, too many missing phrases (942)
fr-FR: 1 missing phrases
it-IT: 1 missing phrases
ja-JP: 1 missing phrases
ko-KR: 1 missing phrases
nl-NL: 1 missing phrases
pl-PL: 90 missing phrases
ru-RU: 7 missing phrases
zh-Hans: 1 missing phrases
zh-Hant: skipped, too many missing phrases (124)
```

**Reply by ThecaTTony** [reply]
es-ar done.

**Reply by DoDiesis** [reply]
🇮🇹 IT done.

**Reply by tsunulukai** [reply]
🇫🇷 / 🇳🇱 👌

**Comment by jcorporation** [comment]
I merged the v20.0.0 branch into devel and updated the [poeditor](https://poeditor.com/projects/view?id=550213) project.

Have fun translating the new terms.

```
bg-BG: skipped, too many missing phrases (1094)
de-DE: fully translated
en-US: fully translated
es-AR: 5 missing phrases
es-ES: skipped, too many missing phrases (961)
es-VE: skipped, too many missing phrases (949)
fi-FI: skipped, too many missing phrases (946)
fr-FR: 5 missing phrases
it-IT: 5 missing phrases
ja-JP: 5 missing phrases
ko-KR: 5 missing phrases
nl-NL: 5 missing phrases
pl-PL: 94 missing phrases
ru-RU: 11 missing phrases
zh-Hans: 5 missing phrases
zh-Hant: skipped, too many missing phrases (128)
```

**Reply by ThecaTTony** [reply]
es-ar done.

**Reply by tsunulukai** [reply]
🇫🇷 / 🇳🇱 👌

**Reply by DoDiesis** [reply]
🇮🇹 IT done.

**Comment by DoDiesis** [comment]
Latest 9 phrases added to Italian translation.

**Reply by ThecaTTony** [reply]
Same for es-ar.

**Reply by tsunulukai** [reply]
Idem for 🇫🇷 / 🇳🇱 👌

**Comment by jcorporation** [comment]
There are new phrases for translation available.

**Reply by ThecaTTony** [reply]
es-ar done.

**Reply by tsunulukai** [reply]
Idem for 🇫🇷 / 🇳🇱 👌

**Reply by DoDiesis** [reply]
🇮🇹 IT done.

**Comment by jcorporation** [comment]
myMPD v22.0.0 is on the way. As for all releases there are some new phrases that must be translated. Many thanks to all translators!

**Reply by tsunulukai** [reply]
🇫🇷 / 🇳🇱 👌

**Reply by DoDiesis** [reply]
🇮🇹 IT done.

**Reply by ThecaTTony** [reply]
es-ar done. Sorry for the delay, it was a busy week.

**Comment by DoDiesis** [comment]
The 4 new strings have been translated to IT 🇮🇹.

**Reply by jcorporation** [reply]
I forgot to update the terms for the latest release. There are now updated and are waiting for translation.

**Reply by ThecaTTony** [reply]
> I forgot to update the terms for the latest release. There are now updated and are waiting for translation.

Any hint on REM term?

https://github.com/jcorporation/myMPD/blob/c407b554e9c3bc82fe3a528ea41f1b3dca67ddd4/src/i18n/json/phrases.json#L738

**Reply by jcorporation** [reply]
That is the CSS unit: rem 	Relative to font-size of the root element. It is the unit for the size 

<img width="1558" height="134" alt="Screenshot From 2025-11-20 15-44-17" src="https://github.com/user-attachments/assets/42576e0a-3bae-4958-ae35-f40a1690c6c9" />

**Reply by DoDiesis** [reply]
> I forgot to update the terms for the latest release. There are now updated and are waiting for translation.

🇮🇹 IT done.

**Reply by tsunulukai** [reply]
FR/NL Done
🇫🇷 / 🇳🇱 👌

**Comment by jcorporation** [comment]
Happy holidays to all contributors!

New terms for the upcoming v24.0.0 are now ready for translation.

**Reply by tsunulukai** [reply]
FR/NL Done
🇫🇷 / 🇳🇱 👌

🎄 Happy holidays y'all ! 🎄

**Reply by ThecaTTony** [reply]
es-ar done.

**Reply by DoDiesis** [reply]
🇮🇹 IT done.

**Comment by jcorporation** [comment]
There are some new terms for translation pending:

```sh
bg-BG: skipped, too many missing phrases (1161)
de-DE: fully translated
en-US: fully translated
es-AR: 4 missing phrases
es-ES: skipped, too many missing phrases (1029)
es-VE: skipped, too many missing phrases (1008)
fi-FI: skipped, too many missing phrases (1005)
fr-FR: 3 missing phrases
it-IT: fully translated
ja-JP: 75 missing phrases
ko-KR: fully translated
nl-NL: 3 missing phrases
pl-PL: 12 missing phrases
ru-RU: 14 missing phrases
zh-Hans: 3 missing phrases
```

**Reply by ThecaTTony** [reply]
es-ar done.

**Reply by tsunulukai** [reply]
FR/NL Done
🇫🇷 / 🇳🇱 👌
